### ISSUE TITLE
Accessibility audit: (Dragon) Say command differs on results between HTML/Nunjucks tab

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#291]
## Describe the issue
'/internal-header' When the dragon user says 'HTML' only the first example HTML tab appears to be brought into focus, and there's no index. Say 'Nunjucks' and Dragin indexes both examples Nunjucks tabs (no focus). 

## Steps to reproduce the issue
Say 'HTML' on '/internal-header'' produces the following:

<img width="292" alt="image" src="https://user-images.githubusercontent.com/809482/143044349-d8c0dc6b-e2a1-413c-aaa4-5ed64a07f7b4.png">

Say 'Nunjucks' on the same view produces the following:

<img width="185" alt="image" src="https://user-images.githubusercontent.com/809482/143044420-1c316bbb-3e7a-4353-98f1-20f0fda92182.png">

## Actual vs expected behaviour
Dragon indexes all the tabs correctly

## Fix Recommendation
- [ ] Check parity of the attributes between the tab elements. If parity matches (they have all the same attributes) then it maybe a page zoom/active viewport issue with Dragon itself.
- [ ] If it's a page soon/active viewport issue, ask for a re-reset at 100% zoom level to rule out
